src/main.rs

```diff
@@ -91,7 +97,26 @@ async fn run(args: Args) -> Result<(), Error> {
                             }
                         }
                     }
-                    "github/release" => {
+                    k if k == GitHubKind::Star.as_str() => {
+                        info!("Backing up repositories for {}", &policy);
```

Comment:
nit: I think it'd be nice to discriminate between owned repos and starred repos.

```suggestion
                        info!("Backing up starred repositories for {}", &policy);
```

Reply:
ack - oversight on my end